### PR TITLE
feat(in-article-cta): done

### PR DESCRIPTION
**Pull request by qiyundai**
Fix:
https://airtable.com/apptk9ROifS4Q03XS/tblzcQd00PWhm85dj/viweDGbEDPbq9hnKi/rec2aQAaWuZq12PMG?blocks=hide

XD:
https://xd.adobe.com/view/8eee2435-edb3-47bf-9271-1fbeeec700c0-dbbd/

Test URL: 
https://in-article-cta--stock--webistry-development.hlx.page/drafts/qiyun/winter-call-for-content-taking-stock-dreaming-big

### CODE REVIEW

**Review by So1ene — APPROVED** [review]

**Comment by rofe** [comment]
@qiyundai @So1ene the test URL is still a 404...

**Comment by So1ene** [comment]
Oops, here you go: https://in-article-cta--stock--webistry-development.hlx.page/drafts/qiyun/winter-call-for-content-taking-stock-dreaming-big

File: pages/blocks/in-article-cta/in-article-cta.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export default async function inArticleCta(block) {
+  const firstLayer = block.querySelector('div');
+
+  block.classList.add('content');
+  firstLayer.classList.add('in-article-cta-container');
+
+  if (firstLayer) {
+    const $divs = block.querySelector('div').querySelectorAll('div');
+
+    Array.from($divs).forEach(($div) => {
```
Comment (rofe) [inline pages/blocks/in-article-cta/in-article-cta.js:10]: Can be simplified to:
```suggestion
    block.querySelector('div').querySelectorAll('div').forEach(($div) => {
```

File: pages/blocks/in-article-cta/in-article-cta.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+export default async function inArticleCta(block) {
+  const firstLayer = block.querySelector('div');
+
+  block.classList.add('content');
```
Comment (rofe) [inline pages/blocks/in-article-cta/in-article-cta.js:4]: Why is this needed?

**Review by rofe — APPROVED** [review]

File: pages/blocks/in-article-cta/in-article-cta.js (hunk 2 above)
Comment (auniverseaway) [reply pages/blocks/in-article-cta/in-article-cta.js:4]: @rofe this is Milo's "default content" class. You get a free container / wrapper max-width with it.
